app/views/step_by_step_pages/schedule.html.erb

```diff
@@ -30,16 +35,30 @@
 <div class="row">
   <div class="col-md-7">
     <%= form_tag do %>
-      <div class="form-group">
-        <%= label_tag "schedule[date][day]", 'Day' %>
-        <%= text_field_tag "schedule[date][day]" %>
-
-        <%= label_tag "schedule[date][month]", 'Month' %>
-        <%= text_field_tag "schedule[date][month]" %>
-
-        <%= label_tag "schedule[date][year]", 'Year' %>
-        <%= text_field_tag "schedule[date][year]" %>
-      </div>
+      <%= render "govuk_publishing_components/components/date_input", {
+        legend_text: legend,
+        name: "schedule[date]",
+        hint: "For example, 01 08 2027",
+        id: "date",
+        error_message: @step_by_step_page.issues[:schedule_date],
```

Comment:
Does adding `error_message` here mean that 

```
<% if @step_by_step_page.errors.any? %>
  <%= render "shared/steps/form_errors", resource: @step_by_step_page %>
<% end %>
```
is no longer needed?

Reply:
Content Publisher displays the error messages both inside the relevant part of the form and at the top of the page. Feels a little odd to me too but I assume is backed up by user research!